conf/app.routes

```diff
@@ -126,3 +126,7 @@ POST        /new-report/event-1-dummy-address                  controllers.addre
 
 GET         /new-report/event-1-employer-payment-nature        controllers.event1.employer.PaymentNatureController.onPageLoad(waypoints: Waypoints ?= EmptyWaypoints)
 POST        /new-report/event-1-employer-payment-nature        controllers.event1.employer.PaymentNatureController.onSubmit(waypoints: Waypoints ?= EmptyWaypoints)
+
+GET        /new-report/member/event-1-benefits-paid-early      controllers.event1.member.BenefitsPaidEarlyController.onPageLoad(waypoints: Waypoints ?= EmptyWaypoints)
```

Comment:
do we need /member/ in both GET and POST ? Can you double check from urls excel please ?

Reply:
Amended in next push, accidentally took URLs from prototype